src/components/molecules/event/eventButton/EventButton.tsx

```diff
@@ -177,9 +177,17 @@ const AuthEventButton: React.FC<AuthButtonProps> = ({
             <div className={styles.cancellationMessageContainer}>
               <p>
                 Arrangementet begynner om under 24 timer, og avmelding så nærme
-                arrangement start vil medføre en merknad. 2 eller flere
-                merknader vil gi nedsatt prioritet på andre arrangementer ut
-                semesteret. Har du gyldig grunn ta kontakt med{' '}
+                arrangement start vil medføre{' '}
+                <u
+                  style={{
+                    textDecorationColor: '#b73653',
+                    textDecorationThickness: '.2rem',
+                  }}>
+                  {' '}
+                  en merknad hvis du har mottat bekreftelse om plass
+                </u>
+                . To eller flere merknader vil gi nedsatt prioritet på andre
+                arrangementer ut semesteret. Har du gyldig grunn ta kontakt med{' '}
                 <a href="mailto:">{event?.host ?? 'post.td.uit.no'}</a>
```

Comment:
I think it's the wrong email here. Try to grep for td.uit.no and see if there is other places we need to change

Reply:
whoops :pencil2: